examples/va/nfv/ovs-dpdk/edpm/openstackdataplanenodeset.yaml

```diff
@@ -0,0 +1,200 @@
+#   OvS DPDK socket memory for your environment
+# - Change "spec.nodeTemplate.ansible.ansibleVars.edpm_ovs_dpdk_memory_channels" to be the desired
+#   OvS DPDK memory channels for your environment
+# - Change "spec.nodeTemplate.ansible.ansibleVars.edpm_ovs_dpdk_vhost_postcopy_support" to be
+#   the desired OvS DPDK vhost postcopy support enabled or not for your environment
+# - Change "spec.nodeTemplate.ansible.ansibleVars.edpm_ovn_bridge_mappings"
+#   to the desired mapping of a bridge with DPDK port name.
+#
+
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: openstack-edpm-ovsdpdk
+spec:
+  env:
+    - name: ANSIBLE_FORCE_COLOR
+      value: "True"
+  services:
+    - repo-setup
+    - download-cache
+    - bootstrap
+    - reboot-os
+    - configure-ovs-dpdk
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+    - ovn
+    - neutron-metadata
+    - libvirt
+    - nova-custom-ovsdpdk
+    - telemetry
+  nodes:
+    edpm-compute-0:
+      ansible:
+        ansibleHost: 192.168.122.100 # CHANGEME
+      hostName: edpm-compute-0
+      networks:
+      - defaultRoute: true
+        fixedIP: 192.168.122.100 # CHANGEME
+        name: CtlPlane
+        subnetName: subnet1
+      - name: InternalApi
+        subnetName: subnet1
+      - name: Storage
+        subnetName: subnet1
+      - name: Tenant
+        subnetName: subnet1
+    edpm-compute-1:
+      ansible:
+        ansibleHost: 192.168.122.101 # CHANGEME
+      hostName: edpm-compute-1
+      networks:
+      - defaultRoute: true
+        fixedIP: 192.168.122.101 # CHANGEME
+        name: CtlPlane
+        subnetName: subnet1
+      - name: InternalApi
+        subnetName: subnet1
+      - name: Storage
+        subnetName: subnet1
+      - name: Tenant
+        subnetName: subnet1
+  networkAttachments:
+    - ctlplane
+  nodeTemplate:
+    ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+    managementNetwork: ctlplane
+    ansible:
+      ansibleUser: cloud-admin
+      ansiblePort: 22
+      ansibleVars:
+        # CHANGEME -- see https://access.redhat.com/solutions/253273
+        # edpm_bootstrap_command: |
+        #       subscription-manager register --username <subscription_manager_username> --password <subscription_manager_password>
+        #       podman login -u <registry_username> -p <registry_password> registry.redhat.io
+        service_net_map:
+          nova_api_network: internal_api
+          nova_libvirt_network: internal_api
+        # CHANGEME
+        timesync_ntp_servers:
+          - hostname: clock.redhat.com
+        # CPU pinning settings
+         # edpm nfv ovs dpdk config
+         edpm_kernel_args: "default_hugepagesz=1GB hugepagesz=1G hugepages=64 iommu=pt intel_iommu=on tsx=off isolcpus=2-11,14-23"  # CHANGEME
+         edpm_tuned_profile: "cpu-partitioning"  # CHANGEME
```

Comment:
We are using this [1] for SRIOV as per the recommendation of the compute team.  Should we use the same here?

[1] https://github.com/openstack-k8s-operators/architecture/blob/b18ae4a41c1bf27d58b15a17d8b7b95db4931ed2/examples/va/nfv/sriov/edpm/values.yaml#L32

Reply:
> We are using this [1] for SRIOV as per the recommendation of the compute team. Should we use the same here?
> 
> [1]
> 
> https://github.com/openstack-k8s-operators/architecture/blob/b18ae4a41c1bf27d58b15a17d8b7b95db4931ed2/examples/va/nfv/sriov/edpm/values.yaml#L32

done